### PR TITLE
Reuploaded: updates for File Services HowTo

### PR DESCRIPTION
**Pull request by jacquesalice**

### CODE REVIEW

File: 04_HowTos/FileService/How_to_use_the_File_Services.html [hunk 1]
```diff
@@ -13092,31 +13093,76 @@
...
+
+</div>
+</div>
+</div>
+<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
+</div><div class="inner_cell">
+<div class="text_cell_render border-box-sizing rendered_html">
+<p><a class="anchor" id="summary"></a></p>
+<h1 id="Summary">Summary<a class="anchor-link" href="#Summary">&#182;</a></h1><p>Files in the virtual storage VOSpace are usually identified via the prefix "<em>vos://</em>". This shorthand identifier is resolved to a user's home directory of the storage space in the service.</p>
+<p>If the "<em>vos://</em>" prefix is instead the name of another user (e.g. "<em>geychaner://</em>"), and the remainder of the path grants public or group read/write access, then the other user's spaces will be accessed. Most user spaces have a "<em>/public</em>" directory to facilitate file sharing (e.g. '<em>geychaner://public/foo.fits</em>' will access the '<em>foo.fits</em>' file from user '<em>geychaner</em>'). Users can make any file (or directory) public by moving it to (or creating a link in) their "/public" directory.</p>
```
Comment (rnikutta) [inline 04_HowTos/FileService/How_to_use_the_File_Services.html:13121]: I propose to use a different demo account here. Glenn Eychaner isn't working at DL anymore. May I suggest "demo00" as the standard strawperson user? I'll send you the password for it in Slack.

File: 04_HowTos/FileService/How_to_use_the_File_Services.ipynb [hunk 2]
```diff
@@ -17,20 +18,61 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a class=\"anchor\" id=\"summary\"></a>\n",
+    "# Summary\n",
+    "\n",
+    "Files in the virtual storage VOSpace are usually identified via the prefix \"*vos://*\". This shorthand identifier is resolved to a user's home directory of the storage space in the service.  \n",
     "\n",
-    "*Revised:  Jan 03, 2019*\n",
+    "If the \"*vos://*\" prefix is instead the name of another user (e.g. \"*geychaner://*\"), and the remainder of the path grants public or group read/write access, then the other user's spaces will be accessed. Most user spaces have a \"*/public*\" directory to facilitate file sharing (e.g. '*geychaner://public/foo.fits*' will access the '*foo.fits*' file from user '*geychaner*'). Users can make any file (or directory) public by moving it to (or creating a link in) their \"/public\" directory.\n",
```
Comment (rnikutta) [inline 04_HowTos/FileService/How_to_use_the_File_Services.ipynb:46]: I propose to use a different demo account here. Glenn Eychaner isn't working at DL anymore. May I suggest "demo00" as the standard strawperson user? I'll send you the password for it in Slack.

Comment (rnikutta) [inline 04_HowTos/FileService/How_to_use_the_File_Services.ipynb:46]: ... will access the ''public/foo.fits'' file from user ...

File: 04_HowTos/FileService/How_to_use_the_File_Services.ipynb [hunk 3]
```diff
@@ -17,20 +18,61 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a class=\"anchor\" id=\"attribution\"></a>\n",
+    "# Disclaimer & Attribution\n",
+    "\n",
+    "If you use this notebook for your published science, please acknowledge the following:\n",
     "\n",
-    "If the \"_vos://_\" prefix is instead the name of another user (e.g. \"_geychaner://_\", and the remainder of the path grants public or group read/write access, then the other user's spaces will be accessed. Most user spaces have a \"_/public_\" directory to facilitate file sharing (e.g. '_geychaner://public/foo.fits_' will access the '_foo.fits_' file from user '_geychaner_'). Users can make any file (or directory) public by moving it to (or creating a link in) their \"/public\" directory.\n",
+    "* Data Lab concept paper: Fitzpatrick et al., \"The NOAO Data Laboratory: a conceptual overview\", SPIE, 9149, 2014, http://dx.doi.org/10.1117/12.2057445\n",
```
Comment (rnikutta) [inline 04_HowTos/FileService/How_to_use_the_File_Services.ipynb:60]: Make both URLs https

File: 04_HowTos/FileService/How_to_use_the_File_Services.ipynb [hunk 4]
```diff
@@ -51,61 +93,107 @@
...
-     "text": [
-      "test2.csv       test3.csv       test6.csv        \n",
-      "\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "print(sc.ls ('demo00://public', format='short'))"
+    "print(sc.ls ('demo00://public',format='short'))"
```
Comment (rnikutta) [inline 04_HowTos/FileService/How_to_use_the_File_Services.ipynb:107]: I see this NB is already using demo00, great!

File: 04_HowTos/FileService/How_to_use_the_File_Services.ipynb [hunk 5]
```diff
@@ -51,61 +93,107 @@
...
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(sc.ls ('sdss_dr14://eboss/spectro/redux'))"
```
Comment (rnikutta) [inline 04_HowTos/FileService/How_to_use_the_File_Services.ipynb:180]: Maybe this line could demonstrate the format='long' argument?

**Review by rnikutta — APPROVED** [review]
Actually, done :)

File: 04_HowTos/FileService/How_to_use_the_File_Services.ipynb (hunk 2 above)
Comment (jacquesalice) [reply 04_HowTos/FileService/How_to_use_the_File_Services.ipynb:46]: Done.

Comment (jacquesalice) [reply 04_HowTos/FileService/How_to_use_the_File_Services.ipynb:46]: Done.

File: 04_HowTos/FileService/How_to_use_the_File_Services.ipynb (hunk 3 above)
Comment (jacquesalice) [reply 04_HowTos/FileService/How_to_use_the_File_Services.ipynb:60]: Done.

**Pull request closed without merging**